content/docs/api-reference/read.md

```diff
@@ -4,11 +4,11 @@ Returns the contents of a tracked file.
 
 ```py
 def open(path: str,
-         repo: str = None,
```

Comment:
that's weird a bit - was it done automatically by the pre-commit hook?

Reply:
I just tried it, and that doesn't seem to be the case. Neither `yarn format-all` nor attempting a dummy commit automatically entered this indent.

Reply:
In fact, our formatting tools seem not to check these code blocks at all.

Reply:
I have `prettier` installed in my IDE. It automatically detects `.prettierrc` and formats everything according to some IDE defaults and setting specified explicitly in `.prettierrc` and `.eslintrc.json`. @rogermparent, there are no specific settings for `.md`, but prettier is still checking these files according to defaults. Anyway, I have returned the formatting.

Reply:
👍